tests/testAutopas/tests/iterators/ParticleIteratorTest.cpp

```diff
@@ -1,193 +1,195 @@
-}
-
-TEST_F(ParticleIteratorTest, testCopyAssignment) {
-  constexpr size_t particlesPerCell = 4;
-  auto cells = IteratorTestHelper::generateCellsWithPattern(10, {1ul, 3ul, 4ul, 7ul, 9ul}, particlesPerCell);
-
-  std::vector<std::vector<Molecule>> additionalVectors(3);
-  additionalVectors[1].emplace_back(Molecule({0., 0., 0.}, {0., 0., 0.}, 1337));
-
-  constexpr bool modifyable = true;
-  autopas::internal::ParticleIterator<Molecule, FMCell, modifyable> iter(
-      &cells, 0, nullptr, IteratorBehavior::ownedOrHalo, &additionalVectors);
-
-  auto iterCopy = iter;
-  iteratorsBehaveEqually(iter, iterCopy);
-}
-
-/**
- * Generates an iterator in a parallel region but iterates with only one and expects to find everything.
- * @note This behavior is needed by VerletClusterLists::updateHaloParticle().
- */
-TEST_F(ParticleIteratorTest, testForceSequential) {
-  constexpr size_t particlesPerCell = 1;
-  auto cells = IteratorTestHelper::generateCellsWithPattern(4, {0ul, 1ul, 2ul, 3ul}, particlesPerCell);
-  auto particlesTotal = cells.size() * particlesPerCell;
-  std::vector<size_t> expectedIndices(particlesTotal);
-  std::iota(expectedIndices.begin(), expectedIndices.end(), 0);
-
-  constexpr size_t numAdditionalVectors = 3;
-  std::vector<std::vector<Molecule>> additionalVectors(numAdditionalVectors);
-
-  size_t particleId = expectedIndices.size() + 100;
-  for (auto &vector : additionalVectors) {
-    vector.emplace_back(Molecule({0., 0., 0.}, {0., 0., 0.}, particleId));
-    expectedIndices.push_back(particleId);
-    ++particleId;
-  }
-
-#pragma omp parallel
-  {
-    std::vector<size_t> foundParticles;
-    constexpr bool modifyable = true;
-    autopas::internal::ParticleIterator<Molecule, FMCell, modifyable> iter(
-        &cells, 0, nullptr, IteratorBehavior::ownedOrHalo | IteratorBehavior::forceSequential, &additionalVectors);
-    for (; iter.isValid(); ++iter) {
-      foundParticles.push_back(iter->getID());
-    }
-    EXPECT_THAT(foundParticles, ::testing::UnorderedElementsAreArray(expectedIndices));
-  }
-}
-
-/**
- * Make sure the iterator does not crash and immediately is marked invalid when offset behind the last cell.
- */
-TEST_F(ParticleIteratorTest, testNothing) {
-  std::vector<FMCell> cells{5};
-  ParticleIterator<Molecule, FMCell, true> iter{&cells, cells.size()};
-  EXPECT_FALSE(iter.isValid());
-}
-
-/**
- * Same as testNothing but also with additional particle vectors.
- */
-TEST_F(ParticleIteratorTest, testNothingWithAdditionalVectors) {
-  std::vector<FMCell> cells{5};
-  std::vector<std::vector<Molecule>> additionalPartilcleVectors(3);
-  ParticleIterator<Molecule, FMCell, true> iter{&cells, cells.size(), nullptr, IteratorBehavior::ownedOrHaloOrDummy,
-                                                &additionalPartilcleVectors};
-  EXPECT_FALSE(iter.isValid());
-}
\ No newline at end of file
+///**
+// TODO: These tests are imho not needed anymore. Please confirm in review.
```

Comment:
I guess most of this is tested by `ContainerIteratorTest`, though I am not quite familiar with the implementation. Therefore just to check:
- Do we test the ContainerIterator with and without additional vectors?
- Do we test `forceSequential` behavior?

Reply:
Seems right. Then we can delete these tests!